### ISSUE TITLE
Logs in CI

### ISSUE THREAD

**Issue by mssun** [mesalock-linux/mkpkg#9]
`mkpkg` does not write logs to CI when successful build a package. (https://ci.mesalock-linux.org/mesalock-linux/mkpkg/26/4)

**Comment by Arcterus** [comment]
When you refer to logs do you mean the ones in the build directory?  If so I don't think those should be dumped by default (maybe unless there's an error).  IMO they should be dumped when the `--verbose` option is given, but this may require disabling parallel building and the progress bars as `--verbose` should probably just write any build output as soon as it's available (like with `makepkg`).

**Comment by Arcterus** [comment]
A short-term solution for this without using `--verbose` would be to just dump the log in `build-all.sh`.

**Comment by mssun** [comment]
No, I mean a simple output showing the current status (which one is started) and build result (successful or failed).

In the CI environment, the progress bar is disabled, we need some meta information similar with the one in the progress bar.
